supervision/keypoint/annotators.py

```diff
@@ -201,6 +202,7 @@ def __init__(
         text_thickness: int = 1,
         text_padding: int = 10,
         border_radius: int = 0,
+        smart_positions: bool = False,
```

Comment:
Let's drop `s` and rename it to `smart_position` as we have other arguments like `text_position`.